packages/core/src/components/panel-stack/_panel-stack.scss

```diff
@@ -13,6 +13,7 @@
   display: flex;
   flex-shrink: 0;
   align-items: center;
+  z-index: 1;
```

Comment:
I thought we were more consistent about using variables like `$pt-z-index-content` [1], but I guess we're not, since I see `z-index: 1` in a few other places... so this is fine

[1] this feels similar to similar to navbar: https://github.com/palantir/blueprint/blob/14ce4ac32291ebd21d5531268b474e1dae55078a/packages/core/src/components/navbar/_navbar.scss#L37